site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -426,18 +426,14 @@ Now, let's create our user. Here, we can specify optional user roles in our appl
 user_registration_request = {
     'registration': {
         'applicationId': application_id,
-        'roles': [
-            'user',
-            'community_helper'
-        ]
+        'roles': []
     },
     'user': {
         'birthDate': '1980-01-01',
         'email': 'jerry@example.com',
         'password': 'password',
         'firstName': 'Jerry',
         'lastName': 'Example',
-        'twoFactorEnabled': False,
```

Comment:
deprecated field

Reply:
That's great. You should probably add a note up at the top that says this has been updated to work with version 1.28 and tested.